### DISCUSSION TITLE
Error with NPM installation on Windows

### DISCUSSION THREAD

**Discussion by KarelRuzicka** [coder/code-server#5568] [Q&A]
While installing code-server on windows using NPM, I have encountered the following error:
`TypeError: '>=' not supported between instances of 'str' and 'int' while trying to load binding.gyp`

Full error dump:
[npm.log](https://github.com/coder/code-server/files/9591919/npm.log)

I have installed all of the required prerequisites:
https://github.com/Microsoft/vscode/wiki/How-to-Contribute#prerequisites

Any idea how to fix this?

**Comment by lg-nineoneninethreenine** [comment]
Firstly, code-server works only on POSIX-compliant systems like macOS(Unix) or Ubuntu(Linux/Unix-like). While Windows uses Windows NT, it is _not_ POSIX-compliant. The only way to install code-server is through WSL.
